### ISSUE TITLE
Has anyone been able to serve this under IIS

### ISSUE THREAD

**Issue by jmbeach** [AHAAAAAAA/PokemonGo-Map#2869]
This isn't an issue, just a question. I want to serve the app under IIS. Has anyone tried this?

**Comment by jangrewe** [comment]
While technically it would be cheating, you can run this *behind* an IIS. But as Python itself is doing the webserver-part, you can't have IIS do the actual work.

**Comment by SethG911** [comment]
Yes, you just need to download the URL rewrite and ARR modules. Overly complicated instructions can be seen [here](http://weblogs.asp.net/owscott/creating-a-reverse-proxy-with-url-rewrite-for-iis). You just need the inbound rewrite, do not apply the outbound or it will fail. Your rule should just be 127.0.0.1:5000.

**Comment by jmbeach** [comment]
@SethG911 Good deal. I'll check that out
